File: src/main/java/genst/Genst.java
package genst;

import cpw.mods.fml.common.Mod;
import cpw.mods.fml.common.event.FMLPostInitializationEvent;
import cpw.mods.fml.common.event.FMLServerStartingEvent;

@Mod(modid = "genst", dependencies = "required-after:lotr")
public class Genst {
	@Mod.EventHandler
	public void postInit(FMLPostInitializationEvent event) {
		GenstMountains.postInit();
		GenstLocations.postInit();
		GenstLogger.postInit();
	}

	@Mod.EventHandler
	public void serverStarting(FMLServerStartingEvent e) {
		e.registerServerCommand(GenstLogger.INSTANCE);
	}
}

File: src/main/java/genst/Genst.kt
package genst

import cpw.mods.fml.common.Mod
import cpw.mods.fml.common.event.FMLPostInitializationEvent
import cpw.mods.fml.common.event.FMLPreInitializationEvent
import genst.world.GenstLocations
import genst.world.GenstMountains
import genst.world.GenstRoads
import genst.world.GenstWaypoints

@Mod(modid = "genst", dependencies = "required-after:lotr", useMetadata = true)
@Suppress("unused", "UNUSED_PARAMETER")
class Genst {
	@Mod.EventHandler
	fun preInit(e: FMLPreInitializationEvent) {
		GenstWaypoints.preInit()
	}

	@Mod.EventHandler
	fun postInit(e: FMLPostInitializationEvent) {
		GenstConfig.postInit()
		GenstMountains.postInit()
		GenstRoads.postInit()
		GenstLocations.postInit()
	}
}